packages/react/src/components/ChoiceGroup/ChoiceGroup.test.tsx

```diff
@@ -207,80 +186,74 @@ describe('ChoiceGroup', () => {
 
   it('returns the current checked option with user interaction', () => {
     const choiceGroupRef = React.createRef<IChoiceGroup>();
-    choiceGroup = mount(<ChoiceGroup options={TEST_OPTIONS} componentRef={choiceGroupRef} />);
+    const { getAllByRole } = render(<ChoiceGroup options={TEST_OPTIONS} componentRef={choiceGroupRef} />);
 
-    const choiceOptions = choiceGroup
-      .getDOMNode()
-      .querySelectorAll(CHOICE_QUERY_SELECTOR) as NodeListOf<HTMLInputElement>;
+    const choiceOptions = getAllByRole('radio') as HTMLInputElement[];
 
     expect(choiceGroupRef.current!.checkedOption).toBeUndefined();
-    ReactTestUtils.Simulate.change(choiceOptions[0]);
+    userEvent.click(choiceOptions[0]);
     expect(choiceGroupRef.current!.checkedOption).toEqual(TEST_OPTIONS[0]);
```

Comment:
Not a new issue but we really ought to verify that the checked state is updated in the DOM, not just in the component state

Reply:
I tried this too and it looks like what's `checked` in the actual DOM *does* get updated, though that may not happen until after a re-render. (The option object doesn't get updated, which is expected.)
https://codepen.io/ecraig12345/pen/mdpMyWO?editors=0011